client/src/components/payment/payment-modal.vue

```diff
@@ -4,7 +4,9 @@
 
       <v-card-title class="d-flex justify-space-between">
         <div>Confirm Name Request</div>
-        <v-icon class="float-right" md color="primary" @click="hideModal()">mdi-close</v-icon>
+        <v-btn icon large class="dialog-close float-right" @click="hideModal()">
```

Comment:
review comments from tracey /bcgov/entity#6137

Reply:
Does this change need to be done in the other modals too?

Reply:
Might require if it has a different size. Tracey mentioned about this modal